docs/includes/topic_f5lbaas-l7_content_switching.rst

```diff
@@ -218,9 +220,9 @@ Configuration
 Further Reading
 ---------------
 
```

Comment:
You're getting that error because the footnote is already defined at the end of the overview section; this one is a duplicate.